method/group/add-pobox.js

```diff
@@ -0,0 +1,75 @@
+const Crut = require('ssb-crut')
+const PoBox = require('../../spec/group/add-pobox')
+
+module.exports = function GroupPobox (ssb, keystore, state) {
+  const poBox = new Crut(ssb, PoBox)
+
+  function isValidRoot (content) {
```

Comment:
@mixmix I couldnt get spec.isRoot/spec.isUpdate to preserve the errorsString..... :( Any tips so i dont have to do it manually??

Reply:
NOTE: reasoning for these two functions

- couldnt use isRoot on the content which it didnt have `poBox: { root: null, previous: null }`, but i still wanted to make sure the staticProps were validated!!

Reply:
LMK if this is okay, or theres a better way to write this!

Reply:
DANG, I forgot about the staticProps if this is an update.
Technicaly I think we should just be using `isUpdate` if we're wanting to crut this later...

I reckon we can ditch the staticProps: root, version

Reply:
Yay, that gets rid of all that code!